CMakeLists.txt

```diff
@@ -121,14 +121,19 @@ cmake_dependent_option(BUILD_NVOF "Build with NVIDIA OPTICAL FLOW SDK support" O
 cmake_dependent_option(BUILD_NVML "Build with NVIDIA Management Library (NVML) support" ON
                        "NOT BUILD_DALI_NODEPS" OFF)
 
+if(${CUDA_VERSION} VERSION_GREATER_EQUAL "12.2")
```

Comment:
On a similar note, we probably can allow user to specify DBUILD_CUFILE with CUDA 11 if on x86?

Reply:
Corrected as well.